crates/holochain_types/src/app.rs

```diff
@@ -84,7 +84,7 @@ pub struct UpdateCoordinatorsPayload {
     pub source: CoordinatorSource,
 }
 
-/// The arguments to create a clone of an existing cell.
+/// The parameters to create a clone of an existing cell.
```

Comment:
They're actually arguments! https://stackoverflow.com/questions/156767/whats-the-difference-between-an-argument-and-a-parameter

Reply:
That would mean that this describes the actual values that are passed in. But this is a definition of a struct. Arguments would be `my_app` and `my_role_name` etc.

Reply:
Maybe just "payload"? :)